fe/fe-core/src/main/java/org/apache/doris/statistics/OlapAnalysisTask.java

```diff
@@ -80,7 +81,8 @@ public void execute() throws Exception {
                     continue;
                 }
                 params.put("partId", String.valueOf(tbl.getPartition(partName).getId()));
-                params.put("partName", String.valueOf(partName));
+                // Avoid error when get the default partition
+                params.put("partName", SqlScanner.isKeyword(partName) ? "`" + partName + "`" : partName);
```

Comment:
Why don't always add backtick, seems this condition is a little redundant

Reply:
it's okay to add together